chart/values.schema.json

```
@@ -5094,6 +5094,53 @@
                     ],
                     "default": null
                 },
+                "hpa": {
+                    "description": "HPA configuration.",
```

Can you be a bit mor verbose for all knowing what HPA is?
```suggestion
                    "description": "Horizontal Pod Autoscaler (HPA) configuration. (optional)",
```

---

Yeah. It should be more details. Add more description in `apiServer.hpa` section in `values.yaml`.